components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { render, screen } from "@testing-library/react";
+import * as navigation from "next/navigation";
+
+import { Footer, FooterItem } from "./Footer";
+
+const usePathname = jest.spyOn(navigation, "usePathname");
+
+const mockPathname = (pathname: string) => {
+  usePathname.mockImplementationOnce(() => pathname);
+};
+
+describe("Footer component ", () => {
+  it("should render", async () => {
+    mockPathname("/");
+    render(<Footer links={[]} />);
+    const footer = await screen.findByTestId("footer");
+
+    expect(footer).toBeInTheDocument();
+  });
+
+  it("should have accesible description", async () => {
+    mockPathname("/");
+    render(<Footer links={[]} />);
+    const footer = await screen.findByTestId("footer");
+
+    expect(footer).toHaveAttribute("aria-label", "Footer");
+  });
+
+  it("should render the logo link", async () => {
+    mockPathname("/foo");
+    render(<Footer links={[]} />);
+    const logo_link = await screen.findByRole("link");
```

Comment:
Can you rename these variables to camelCase?

Reply:
Ok.

Reply:
Done.